session/session_test.go

```diff
@@ -2493,6 +2493,22 @@ func (s *testSessionSuite2) TestCommitRetryCount(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *testSessionSuite2) TestEnablePartition(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
```

Comment:
`select @@tidb_enable_table_partition` here returns off?

Reply:
Thanks, I will set the initial value for the upgradings.